core/src/main/scala/org/http4s/Message.scala

```diff
@@ -187,6 +187,15 @@ sealed abstract case class Request[F[_]](
       attributes = attributes
     )
 
+  def mapK[G[_]](f: F ~> G): Request[G] = Request[G](
```

Comment:
Isn't this just `withBodyStream`?

Reply:
No, withBodyStream requires the same `F` although I might question that decision as well.

https://github.com/http4s/http4s/blob/6339d7cb3bee969502a6b50d7989853a9bf57a50/core/src/main/scala/org/http4s/Message.scala#L79

Reply:
Oh, good point.